go/signedexchange/certurl/cert_chain.go

```diff
@@ -0,0 +1,66 @@
+package certurl
```

Comment:
`certchain.go` is better than `cert_chain.go`, since `_` has a special meaning in filenames in Go to indicate platforms (e.g. `foo_windows.go` is compiled only for Windows)

Reply:
Done.